neo4j/auth/cert.go

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [https://neo4j.com]
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package auth
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"os"
+	"sync"
+)
+
+// ClientCertificate holds paths to a TLS certificate file and its corresponding private key file.
+// This struct is used to load certificate-key pairs for use in TLS connections.
```

Comment:
```suggestion
// This struct is used to load client certificate-key pairs for use in mutual TLS.
```
Or something along the lines of this. I reckon having the keyword mTLS or mutual TLS in here will increase discoverability and clarity of the docs.

Reply:
Agree 😄